app/src/main/java/com/keylesspalace/tusky/adapter/StatusBaseViewHolder.java

```diff
@@ -198,23 +197,21 @@ protected void setSpoilerAndContent(boolean expanded,
                 if (getAdapterPosition() != RecyclerView.NO_POSITION) {
                     listener.onExpandedChange(isChecked, getAdapterPosition());
                 }
-                this.setTextVisible(isChecked, content, mentions, emojis, poll, listener);
+                this.setTextVisible(isChecked, content, mentions, emojis, poll, statusDisplayOptions, listener);
             });
-            this.setTextVisible(expanded, content, mentions, emojis, poll, listener);
+            this.setTextVisible(expanded, content, mentions, emojis, poll, statusDisplayOptions, listener);
         }
     }
 
-    private void setTextVisible(boolean expanded,
-                                Spanned content,
-                                Status.Mention[] mentions,
-                                List<Emoji> emojis,
-                                @Nullable PollViewData poll,
+    private void setTextVisible(boolean expanded, Spanned content, Status.Mention[] mentions,
+                                List<Emoji> emojis, @Nullable PollViewData poll,
+                                StatusDisplayOptions statusDisplayOptions,
```

Comment:
I would prefer one arg per line

Reply:
Oops, that's what Intellij did when I moved arguments order I guess